### ISSUE TITLE
The new Blazor template with minimal API breaks identity scaffolding

### ISSUE THREAD

**Issue by AlbertoPa** [dotnet/aspnetcore#35353]
With the release of Net 6 Preview 7, templates for Blazor have been simplified to implement the new "minimal API" approach. While this serves to demonstrate the new functionality, they also make some key functionality in Visual Studio not working properly. 

Specifically, the blazor hosted app with identity server has minimal API enabled, which prevents the scaffolder from generating identity pages for customization.
The user finds out that scaffolding identity pages in this template is not possible only after selecting the pages to scaffold and attempting the operation. 

P. S. I think it would have been better to demonstrate the minimal API functionality in a separate template for the API only, leaving the Blazor templates unchanged. To the very least the full API experience should be left as option (i.e., a template for Blazor with minimal API and one with full API to ensure complete functionality), rather than forcing developers to recode what was there in the first place until preview 6.

**Comment by davidfowl** [comment]
This is a known issue and is should be fixed in the next version of the tooling.

cc @vijayrkn

**Comment by vijayrkn** [comment]
The identity fix is very close to getting checked-in. 

/cc @deepchoudhery

**Comment by AlbertoPa** [comment]
Thanks!

**Comment by SteveSandersonMS** [comment]
@vijayrkn Thanks for answering. Is there another issue I can close this as a duplicate of? Or is it tracked somewhere else and I can just close this?

**Comment by vijayrkn** [comment]
@SteveSandersonMS - You can dupe it against this bug - https://github.com/dotnet/Scaffolding/issues/1584
